packages/frontend-2/components/preview/Image.vue

```diff
@@ -102,6 +102,7 @@ const finalPreviewTransitioner = ref(
   null as Nullable<{ triggerTransition: () => Promise<void> }>
 )
 
+const { $isAppHydrated } = useNuxtApp()
```

Comment:
super annoying! there are preview image related mismatches only happening in prod, and while i've dealt w/ most of them, some still remain